lib/analytics_data.rb

```diff
@@ -0,0 +1,58 @@
+class AnalyticsData
+  def initialize(elasticsearch_url, indices)
+    @elasticsearch_url = elasticsearch_url
+    @indices = indices
+  end
+
+  def headers
+    %w(
+      ga:productSku
+      ga:productName
+      ga:productBrand
+      ga:productCategoryHierarchy
+      ga:dimension72
+      ga:dimension73
+      ga:dimension74
+      ga:dimension75
+      ga:dimension76
+      ga:dimension77
+      ga:dimension78
+      ga:dimension79
+      ga:dimension80
```

Comment:
I've refactored the class so that the CSV headers and values are defined together. This makes their purpose clearer (I've added comments on the placeholders) and should stop them from getting out of sync.